### ISSUE TITLE
Add an about page to the app where people can version, creator, repository etc.

### ISSUE THREAD

**Issue by FriederHannenheim** [moritz-weber/mucke#105]
This would also help knowing which version you have when reporting issues. I think the best place for this would be the settings page

**Comment by moritz-weber** [comment]
Yes, good idea. I also think, the settings are the right place for this.

I wouldn't put information about the creator there, as I don't think this is useful for the user. And with more contributors, "creator" becomes fuzzy anyway. 

App version, a link to the repository (maybe also one to open an issue), and a link to Weblate would be very useful, though. Perhaps, I could also activate discussions for the repository, so users can discuss ideas/issues there.

For the app info, you can look at the new ImportExportRepositoryImpl.
